test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -916,16 +907,28 @@ def test_cgroup_offline_node(self):
         self.logger.info("tasks: %s" % tasks)
         # Write each PID into the tasks file for the freezer cgroup
         for task in tasks[1:]:
+            cmd = ['/bin/echo', '%s' % task]
             with open(os.path.join(fdir, 'tasks'), 'w') as fd:
-                fd.write(str(task) + '\n')
+                ret = self.du.run_cmd(self.mom.hostname, cmd,
```

Comment:
This should be updated to use mkstemp and above os.mkdir should be updated to use du.mkdir and run as root.